superbench/runner/runner.py

```diff
@@ -265,6 +265,33 @@ def __create_single_node_summary(self, node_path):    # pragma: no cover # noqa:
 
         return results_summary
 
+    def __generate_metric_name(self, benchmark_name, metric, rank_count, run_count, curr_rank, curr_run):
```

Comment:
I think we would like to remove the data from other ranks. Why do we need this function.

Reply:
single-gpu test run on multiple gpus? need to differentiate different gpus by ranks in this case

Reply:
Correct, this function can work for all benchmarks, take kernel-launch for example, we can have
kernel-launch/overhead_wall:0
kernel-launch/overhead_wall:1
kernel-launch/overhead_wall:2
....